spec/requests/users_controller_spec.rb

```diff
@@ -96,6 +96,13 @@
         expect(response).to redirect_to("/account/sign-in")
       end
     end
+
+    describe "#resend_invite" do
+      it "does not allow resending activation emails" do
+        get "/users/#{user.id}/deactivate", headers: headers, params: {}
```

Comment:
I know it's largely the done thing in this project, but it is best practice to also use _path (eg here would it be `deactivate_user_path` ? ) helpers in spec files as you are then also implicitly testing your routing.

Reply:
All absolute paths changed to paths in upcoming commit